plexpy/notification_handler.py

```diff
@@ -187,7 +188,13 @@ def notify_conditions(notify_action=None, stream_data=None, timeline_data=None,
                 user_sessions = [s for s in result['sessions'] if s['user_id'] == stream_data['user_id']]
 
             if plexpy.CONFIG.NOTIFY_CONCURRENT_BY_IP:
-                evaluated = len(Counter(s['ip_address'] for s in user_sessions)) >= plexpy.CONFIG.NOTIFY_CONCURRENT_THRESHOLD
+                ip_addresses = []
+                for s in user_sessions:
+                    if helpers.ip_type(s['ip_address']) == 'IPv6':
+                        ip_addresses.append(helpers.get_ipv6_network_address(s['ip_address']))
+                    if helpers.ip_type(s['ip_address']) == 'IPv4':
+                        ip_addresses.append(s['ip_address'])
+                evaluated = len(Counter(ip_addresses)) >= plexpy.CONFIG.NOTIFY_CONCURRENT_THRESHOLD
```

Comment:
I think this can be a `set()` instead.

```suggestion
                ip_addresses = set()
                for s in user_sessions:
                    if helpers.ip_type(s['ip_address']) == 'IPv6':
                        ip_addresses.add(helpers.get_ipv6_network_address(s['ip_address']))
                    elif helpers.ip_type(s['ip_address']) == 'IPv4':
                        ip_addresses.add(s['ip_address'])
                evaluated = len(ip_addresses) >= plexpy.CONFIG.NOTIFY_CONCURRENT_THRESHOLD
```